stacker/config/__init__.py

```diff
@@ -3,7 +3,10 @@
 
 import yaml
 
-from . import exceptions
+from .. import exceptions
+
+# register translators (yaml constructors)
+from .translators import *  # NOQA
```

Comment:
yea, me either. this triggers the registration of the yaml constructors though. otherwise we would be importing something we didn't need. i could do something like within translators we append the constructors we want registered to a list, import that list and then register them with yaml within the config.